orttraining/tools/ci_test/run_batch_size_test.py

```diff
@@ -46,8 +46,8 @@ def main():
     ]
 
     configs['MI100_32G'] = [
-        Config(True, 128, 200, 20, ""),
-        Config(True, 512, 30, 80, ""),
+        Config(True, 128, 192, 20, ""),
+        Config(True, 504, 30, 80, ""),
```

Comment:
>504 [](http://example.com/codeflow?start=21&length=3)

we should keep this 512, and change 30->xx

Reply:
Made the change and validated it on the pipeline.